magento/pwa-studio
# Requesting price summary fragment in shipping info mutation.

## Description

The price summary is not updated on submitting shipping information on the checkout page. This is because the price summary is not being fetched in the mutation result.

Solved by adding price summary fragment to shipping information mutation.

## Related Issue
Closes [PWA-614](https://jira.corp.magento.com/browse/PWA-614)

### Verification Stakeholders
@tjwiebell 

### Verification Steps
1. Go to the deployed app's checkout page.
2. Update the shipping address to a state that has no tax (Texas). There should be no tax section in the price summary.
3. Update the shipping address to a state that has tax (California). There should be a tax section in the price summary.

@fooman mentioned there is a setting in the admin panel to perform tax calculation based on billing address instead. @dpatil-magento when you are testing, try to check for that option as well. The code has logic to render tax changes on billing address change as well.

## Screenshots / Screen Captures (if appropriate)

Lone Star State (No Tax)
![image](https://user-images.githubusercontent.com/35203638/83453818-c8068280-a420-11ea-87e2-58d67e87e9f4.png)

Golden State (😮 Tax)
![image](https://user-images.githubusercontent.com/35203638/83453899-ecfaf580-a420-11ea-85a5-e0d309b12555.png)

## Checklist
No tests needed. It is a GQL update.
